core/src/allocation.jl

```diff
@@ -883,20 +883,20 @@ function set_objective_priority!(
         add_user_term!(ex, edge_id, objective_type, d, allocation_model)
     end
 
-    # Add flow cost
-    if objective_type == :linear_absolute
-        cost_per_flow = 0.5 / length(F)
-        for flow in F
-            JuMP.add_to_expression!(ex, cost_per_flow * flow)
-        end
-    elseif objective_type == :linear_relative
-        if demand_max > 0.0
-            cost_per_flow = 0.5 / (demand_max * length(F))
-            for flow in F
-                JuMP.add_to_expression!(ex, cost_per_flow * flow)
-            end
-        end
-    end
+    # # Add flow cost
+    # if objective_type == :linear_absolute
+    #     cost_per_flow = 0.5 / length(F)
+    #     for flow in F
+    #         JuMP.add_to_expression!(ex, cost_per_flow * flow) #
+    #     end
+    # elseif objective_type == :linear_relative
+    #     if demand_max > 0.0
+    #         cost_per_flow = 0.5 / (demand_max * length(F))
+    #         for flow in F
+    #             JuMP.add_to_expression!(ex, cost_per_flow * flow) #
+    #         end
+    #     end
+    # end
```

Comment:
Don't forget to remove commented out code before merging. Also, note that `demand_max` is no longer needed without the flow costs so you can remove the initialization and computation of that variable. 

Our collegues are quite strict with removing unneeded code. If a variable shows up grey in the code, that usually means that it is not used and thus that you can remove it.

Reply:
Yes! I left it here for other purpose. I will remove it before merging